simulation/traffic_simulator/include/traffic_simulator/api/api.hpp

```diff
@@ -309,30 +309,22 @@ class API
   FORWARD_TO_ENTITY_MANAGER(entityExists);
   FORWARD_TO_ENTITY_MANAGER(getBehaviorParameter);
   FORWARD_TO_ENTITY_MANAGER(getBoundingBox);
-  FORWARD_TO_ENTITY_MANAGER(getBoundingBoxDistance);
-  FORWARD_TO_ENTITY_MANAGER(getBoundingBoxLaneLateralDistance);
-  FORWARD_TO_ENTITY_MANAGER(getBoundingBoxLaneLongitudinalDistance);
-  FORWARD_TO_ENTITY_MANAGER(getBoundingBoxRelativePose);
   FORWARD_TO_ENTITY_MANAGER(getConventionalTrafficLight);
   FORWARD_TO_ENTITY_MANAGER(getConventionalTrafficLights);
   FORWARD_TO_ENTITY_MANAGER(getCurrentAccel);
   FORWARD_TO_ENTITY_MANAGER(getCurrentAction);
   FORWARD_TO_ENTITY_MANAGER(getCurrentTwist);
-  FORWARD_TO_ENTITY_MANAGER(getDistanceToLaneBound);
-  FORWARD_TO_ENTITY_MANAGER(getDistanceToLeftLaneBound);
-  FORWARD_TO_ENTITY_MANAGER(getDistanceToRightLaneBound);
   FORWARD_TO_ENTITY_MANAGER(getEgoName);
+  FORWARD_TO_ENTITY_MANAGER(getEntity);
   FORWARD_TO_ENTITY_MANAGER(getEntityNames);
   FORWARD_TO_ENTITY_MANAGER(getEntityStatus);
   FORWARD_TO_ENTITY_MANAGER(getEntityStatusBeforeUpdate);
+  FORWARD_TO_ENTITY_MANAGER(getHdmapUtils);
   FORWARD_TO_ENTITY_MANAGER(getLaneletLength);
   FORWARD_TO_ENTITY_MANAGER(getLaneletPose);
-  FORWARD_TO_ENTITY_MANAGER(getLateralDistance);
   FORWARD_TO_ENTITY_MANAGER(getLinearJerk);
-  FORWARD_TO_ENTITY_MANAGER(getLongitudinalDistance);
   FORWARD_TO_ENTITY_MANAGER(getMapPose);
   FORWARD_TO_ENTITY_MANAGER(getMapPoseFromRelativePose);
-  FORWARD_TO_ENTITY_MANAGER(getRelativePose);
```

Comment:
Please add Destructive Changes section in pull request description and, add migration guide for all deprecated APIs.

Reply:
@hakuturu583 
I've updated the PR description, please let me know if this is sufficient.